lightning/src/ln/channelmanager.rs

```diff
@@ -574,6 +577,9 @@ pub struct ChannelDetails {
 	/// True if the channel is (a) confirmed and funding_locked messages have been exchanged, (b)
 	/// the peer is connected, and (c) no monitor update failure is pending resolution.
 	pub is_live: bool,
+
+	/// Counterparty forwarding information.
```

Comment:
nit: could maybe be expanded to say, like, "information on the fees and requirements that the counterparty requires when forwarding payments to us through this channel".